triagebot.toml

```diff
@@ -976,8 +976,8 @@ warn_non_default_branch = true
 contributing_url = "https://rustc-dev-guide.rust-lang.org/getting-started.html"
 users_on_vacation = [
     "jieyouxu",
-    "jyn514",
```

Comment:
They don't seem to appear anywhere in this file, so I removed them.

Reply:
jyn is on alumni status so yeah

Reply:
I think that jyn wanted to remain there to avoid rustbot pings, IIRC, as some people were still pinging jyn months after they stopped actively working on the Project.

(somewhat ironically) @jyn514 Do you still want to be in this list?

Reply:
Does rustbot ping people that aren't in the triagebot file? That sounds weird :thinking:

Reply:
Oh right good point, I think people can still r? jyn otherwise?

Reply:
at the time i left you couldn’t assign PRs to people outside the org. if that’s still the case i'm happy for you to remove me from this list.